pyquil/api/_compiler.py

```diff
@@ -352,7 +347,8 @@ def native_quil_to_executable(self, nq_program: Program) -> PyQuilExecutableResp
     @_record_call
     def reset(self):
         """
-        Reset the state of the QPUCompiler Client connections.
+        Reset the state of the QVMCompiler quilc connection
         """
-        pyquil_config = PyquilConfig()
-        self.client = refresh_client(self.client, pyquil_config.quilc_url)
+        timeout = self.client.timeout
+        self.client.close()
+        self.client = Client(self.endpoint, timeout=timeout)
```

Comment:
```suggestion
        self.client = Client(self.client.endpoint, timeout=timeout)
```

guessing here

Reply:
should be `self.endpoint`, but that wasn't being persisted. thanks